erts/doc/src/erl_cmd.xml

```diff
@@ -1035,6 +1035,40 @@ $ <input>erl \
           descriptor input events.
         </p>
       </item>
+      <tag><marker id="+JPcover"/><c>+JPcover true|false|function|line_coverage|line_counters</c></tag>
+      <item>
+        <p>Since: OTP 27.0</p>
+        <p>Enables or disables support for coverage when running with
+        the JIT. Defaults to false.</p>
+        <taglist>
+          <tag><c>function</c></tag>
```

Comment:
`function_coverage`? Should there be a `function_counter`? or are we stepping too much on the toes of `call_count` tracing now?

Reply:
Maybe `line_coverage` should be renamed `line`?

Reply:
Yes, I think this should add `function_counters` for symmetry (and rename `line_coverage` to `line`). It is overlapping functionality with `call_count` tracing and `cprof`, but there are some differences. `call_count` tracing can be disabled without reloading the module, but `function_counters` should have less overhead.